README.md

```diff
@@ -32,14 +32,42 @@ pub struct AppInfo {
 ```
 
 ### Setting Up the Bot
+
 ### Requirements
-You need to have `rust` and `node` installed.
 
+You need to have `rust` and `node` installed.
+Optionally, have [pnpm](https://pnpm.io/) installed.
 
 ### Setup
-1. Go to the `frontend` folder and install the packages using `pnpm i`.
-2. Build the frontend using `pnpm build`.
-3. Start the bot (from the root folder) `addr="<email>" mail_pw="<password>" cargo r`
+
+Go to the `frontend` folder and install the packages using `pnpm install` or `pnpm i`.
+If you do not have `pnpm` installed, run `npx pnpm install` instead.
```

Comment:
We should maybe not insist on people using pnpm as they can eaqually good just use npm. Myabe Just give a hint that pnpm is probably better because it usese the lock file, but otherwise instructions npm can be used.